devito/tools/utils.py

```diff
@@ -75,6 +75,13 @@ def is_integer(value):
     return isinstance(value, (int, np.integer, sympy.Integer))
 
 
+def contains_val(val, items):
```

Comment:
can we just call this "contains" ?

Reply:
I thought `contains` was a bit generic and might conflict with some default exisiting functions but happy to change

Reply:
So there is not any better way for this? weird

Reply:
Depends what you call "better". The previous was `val in as_tuple(items)` but this can be really really bad if you have something like `items = range(0, 1e6)` because it will evaluate the `range` as a super long tuple.

Reply:
OK fine by me